### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by apexad** [ToddGreenfield/homebridge-directv#4]
```bash
Apr 22 08:05:40 raspberrypi homebridge[16110]: [4/22/2017, 8:05:40 AM] [DIRECTV] Unable to call for current channel.
Apr 22 08:05:40 raspberrypi homebridge[16110]: [4/22/2017, 8:05:40 AM] [DIRECTV] Current Channel is 0
Apr 22 08:05:40 raspberrypi homebridge[16110]: /usr/lib/node_modules/homebridge/node_modules/hap-nodejs/lib/util/once.js:12
Apr 22 08:05:40 raspberrypi homebridge[16110]: throw new Error("This callback function has already been called by someone else; it can only be called one time.");
Apr 22 08:05:40 raspberrypi homebridge[16110]: ^
Apr 22 08:05:40 raspberrypi homebridge[16110]: Error: This callback function has already been called by someone else; it can only be called one time.
Apr 22 08:05:40 raspberrypi homebridge[16110]: at /usr/lib/node_modules/homebridge/node_modules/hap-nodejs/lib/util/once.js:12:13
Apr 22 08:05:40 raspberrypi homebridge[16110]: at /usr/lib/node_modules/homebridge-directv/index.js:101:7
Apr 22 08:05:40 raspberrypi homebridge[16110]: at /usr/lib/node_modules/homebridge-directv/node_modules/directv-remote/index.js:264:21
Apr 22 08:05:40 raspberrypi homebridge[16110]: at IncomingMessage.<anonymous> (/usr/lib/node_modules/homebridge-directv/node_modules/directv-remote/index.js:311:17)
Apr 22 08:05:40 raspberrypi homebridge[16110]: at emitNone (events.js:91:20)
Apr 22 08:05:40 raspberrypi homebridge[16110]: at IncomingMessage.emit (events.js:185:7)
Apr 22 08:05:40 raspberrypi homebridge[16110]: at endReadableNT (_stream_readable.js:974:12)
Apr 22 08:05:40 raspberrypi homebridge[16110]: at _combinedTickCallback (internal/process/next_tick.js:74:11)
Apr 22 08:05:40 raspberrypi homebridge[16110]: at process._tickCallback (internal/process/next_tick.js:98:9)
```

**Comment by apexad** [comment]
Right now I just deleted the set Channel scenes and it seems to be stable again. I will look into this issue soon and put up a PR if I am able to fix this.

I am thinking that it will be something similar to the other PR I just put up where getChannel actually checks if the box is on before running getTuned does not run if it is off and instead it just reports it is on channel 0.

**Comment by ToddGreenfield** [comment]
Feel free to do a PR for this as well as #3 if you would like- thanks for the contrib-

**Comment by apexad** [comment]
As I look into this a little bit it may have actually been an issue with having multiple channel scenes.

**Comment by ToddGreenfield** [comment]
This is happening on your Geni (non-primary) boxes, correct? I believe I have this resolved and will upload a fix tomorrow sometime.

**Comment by apexad** [comment]
Yeah, I think it was just on the non Primary boxes.

**Comment by ToddGreenfield** [comment]
Fixed in 0.0.9.
